.gitignore

```diff
@@ -2,6 +2,7 @@
 .DS_Store
 npm-debug.log*
 node_modules
+.yarn
```

Comment:
?

Reply:
This is yarn cache. Felt like adding it here since there's going to be a good number of people using Yarn.